docs/resources/azure_streaming_analytics_functions.md

```diff
@@ -0,0 +1,84 @@
+---
+title: About the azure_streaming_analytics_functions Resource
+platform: azure
+---
+
+# azure_streaming_analytics_functions
+
+Use the `azure_streaming_analytics_functions` InSpec audit resource to test properties and configuration of multiple Azure streaming analytics fuctions.
+
+## Azure REST API version, endpoint and http client parameters
+
+This resource interacts with api versions supported by the resource provider.
+The `api_version` can be defined as a resource parameter.
+If not provided, the latest version will be used.
+For more information, refer to [`azure_generic_resource`](azure_generic_resource.md).
+
+Unless defined, `azure_cloud` global endpoint, and default values for the http client will be used.
+For more information, refer to the resource pack [README](../../README.md).
+
+## Availability
+
+### Installation
+
+This resource is available in the [InSpec Azure resource pack](https://github.com/inspec/inspec-azure).
+For an example `inspec.yml` file and how to set up your Azure credentials, refer to resource pack [README](../../README.md#Service-Principal).
+
+## Syntax
+
+An `azure_streaming_analytics_functions` resource block returns all functions  under a job.
+```ruby
+
+describe azure_streaming_analytics_functions(resource_group: "EmptyExampleGroup", job_name: "azure_streaming_job_name") do
+  #...
+end
+
+```
+or
+
+## Parameters
+
+- `resource_group` (Optional)
```

Comment:
Should this be both resource_group and job_name? Are both optional or required?

Reply:
both are required in plural resource Updated the doc